Meetings/Meeting 2 Notes #94002408

```diff
@@ -0,0 +1,32 @@
+Team D - Meeting 5/2/15
+Locaiton: Library
+
+
+Team Members Present
+
+	  1) Levi Schoen
+		2) Joseph Stewart
+		3) Brandon Isom
+		4) Raven Allan
+		5) Laura DeWitt
+		6) Brady Sullivan
+		7) Parker Berger (note taker)
+
+
+Agenda Item
+
+	1. Picking a project after presentations
+
```

Comment:
@parkerBerger would've liked to seen each team members top 3 here, and the vote tally for the top 7 projects. A little more detail in the future would be awesome! Thanks for getting these posted :+1: